src/v2/Apps/Collect/Routes/Collection/Components/Header/index.tsx

```diff
@@ -228,16 +227,16 @@ export const CollectionHeader: FC<Props> = ({ artworks, collection }) => {
 
                       <Spacer mt={1} />
 
-                      <Serif size={["6", "10"]} element="h1">
+                      <Text variant="largeTitle" as="h1">
```

Comment:
This is the idiomatic way to make semantic headings out of Text, yes @dzucconi ?

Reply:
Yep! 👍